src/workerd/io/container.capnp

```diff
@@ -34,6 +34,12 @@ interface Container @0x9aaceefc06523bca {
     # If null, the container will start with the environment variables defined in its image.
     # The format is defined as a list of `NAME=VALUE`.
     # The container runtime should validate the environment variables input.
+
+    hardTimeoutMs @3 :UInt32;
```

Comment:
Why is this a `:Uint32`?  (Yes, 2**32ms is 49 days, but I don't see the point of having a shorter type here.)